frontend/views/containers/proposals/ProposalBox.vue

```diff
@@ -42,17 +53,16 @@ export default {
     title () {
       const { identityContractID } = this.proposal.meta
       const username = this.$store.state[identityContractID].attributes.name
-      const currentUsername = this.ourUserIdentityContract.attributes.name
-      const who = username === currentUsername ? L('You') : username
+      const who = username === this.ourUsername ? L('You') : username
       const isAnyOpen = this.proposalHashes.some(hash => this.currentGroupState.proposals[hash].status === STATUS_OPEN)
 
       if (!isAnyOpen) {
-        return L('{who} proposed', { who })
+        return L('{who}{_strong} proposed:', { who, ...this.LTags('strong') })
       }
 
-      return username === currentUsername
-        ? L('You are proposing')
-        : L('{who} is proposing', { who })
+      return username === this.ourUsername
+        ? L('You are proposing:')
+        : L('{strong_}{who}{_strong} is proposing:', { who, ...this.LTags('strong') })
```

Comment:
See same [feedback to @pieer](https://github.com/okTurtles/group-income-simple/pull/724#discussion_r338022233), copied here for convenience:

The pattern: `{tag_}{variable}{_tag}` should be rewritten using just `{variable}` to make life less confusing for the translators. Tags are useful when the thing between them is a word to be translated, i.e. `{tag_}english word or fragment that is not a variable{_tag}`.

So here you can do:

```
        i18n(
          v-if='needsIncome'
          tag='p'
          data-test='headerNeed'
          :args='{ amount: `<span class="has-text-bold has-text-0">${upTo}</span>` }'
        ) You need {amount}
```

The same goes for the two `i18n`s below and any other ones in this PR that follow the pattern `{tag_}{variable}{_tag}`

Reply:
So.. I didn't understand exactly when we should use `{tag_}` over `<strong>` in `args`. Can you clarify that?

Reply:
Let's make sure we're talking about the same thing, because the L function doesn't have anything called `args`, while `i18n` does.

In i18n, variables are passed through `:args=`, and in the `L` function, they are passed as the second parameter.

String literals (the stuff that's given to translators), must never contain HTML.

Here we have a variable that translators can move around, `{move}`. It doesn't need the variables `{strong_}` and `{_string}` around it, because that's redundant. IF instead we had something like `{who} some text` that needed to be made bold as a whole unit, THEN we can introduce `{strong_}` and `{_strong}` around it, like so: `{strong_}{who} some text{_strong}`.

But in the case of the variable itself needing to be bolded, there is absolutely no reason to bother the translators by introducing more variables than there needs to be. That is the case here. We only need `{who}` in here. The fact that the value is surrounded by HTML `<strong>` doesn't need to be evident in the string-to-be-translated. So here it should be removed, and placed into the value-part. In this case:

```
{ who: `<strong>${who}</strong>` }
```

Reply:
Gotcha. So, in summary, use tags when they wrap variables (who) and raw text. Otherwise, write the html directly as part of the variable value, so translators don't need to bother with extra information. I'll add it to our i18n docs.

Reply:
> use tags when they wrap variables (who) and raw text

(or just raw text, yes)

> Otherwise, write the html directly as part of the variable value, so translators don't need to bother with extra information. I'll add it to our i18n docs.

😄 👍